### PR TITLE
Query system tables with influxctl

### PR DESCRIPTION
**Pull request by sanderson**
Related to https://github.com/influxdata/influxctl/pull/321

This adds information about querying system tables with `influxctl`.

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/admin/cluster-data.md [hunk 1]
```diff
@@ -0,0 +1,398 @@
+---
+title: Query cluster system data
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/cluster-data.md:2]: When I first see this, I take it to mean _Query cluster_ system data. 🤷

File: content/influxdb/cloud-dedicated/admin/cluster-data.md [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+performance of your {{< product-name omit=" Clustered" >}} cluster.
+
+<!--------------- UPDATE THE DATE BELOW AS EXAMPLES ARE UPDATED --------------->
+
+#### System tables are subject to change
+
+System tables are not part of InfluxDB's stable API and may change with new releases.
+The provided schema information and query examples are valid as of **April 11, 2024**.
+If you detect a schema change or a non-functioning query example, please
+[let us know](https://github.com/influxdata/docs-v2/issues/new/choose).
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/cluster-data.md:34]:
```suggestion
[submit an issue](https://github.com/influxdata/docs-v2/issues/new/choose).
```

File: content/influxdb/cloud-dedicated/admin/cluster-data.md [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+{{% /warn %}}
+
+## Query system tables
+
+{{% warn %}}
+_Querying system tables [may impact overall cluster performance](#may-impact-overall-cluster-performance)._
+{{% /warn %}}
+
+{{% note %}}
+Querying system tables requires **`influxctl` v2.8.0 or newer**.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/cluster-data.md:46]: But, it doesn't require `influxctl`, right? You can still use the `iox-debug` header?

File: content/influxdb/cloud-dedicated/admin/cluster-data.md [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+SELECT
+  table_name,
+  SUM(total_size_mb) AS total_size_mb
+FROM
+  system.partitions
+GROUP BY
+  table_name
+```
+
+#### View the size of a specific tables in megabytes
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/cluster-data.md:329]:
```suggestion
#### View the size of a specific table in megabytes
```

**Review by jstirnaman — APPROVED** [review]
Minor fix and some questions.

File: content/influxdb/cloud-dedicated/admin/cluster-data.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/admin/cluster-data.md:2]: Ah, ok. I can see that. What about just "Query system data"?

File: content/influxdb/cloud-dedicated/admin/cluster-data.md (hunk 3 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/admin/cluster-data.md:46]: True. What about:

```suggestion
Querying system tables with `influxctl` requires **`influxctl` v2.8.0 or newer**.
```

File: content/influxdb/cloud-dedicated/admin/cluster-data.md [hunk 5]
```diff
@@ -0,0 +1,398 @@
...
+  - [View query logs for queries with end-to-end durations above a threshold](#view-query-logs-for-queries-with-end-to-end-durations-above-a-threshold)
+- [Partitions](#partitions)
+  - [View partition templates of all tables](#view-partition-templates-of-all-tables)
+  - [View the partition template of a specific table](#view-the-partition-template-of-a-specific-table)
+  - [View all partitions for a table](#view-all-partitions-for-a-table)
+  - [View the number of partitions per table](#view-the-number-of-partitions-per-table)
+  - [View the number of partitions for a specific table](#view-the-number-of-partitions-for-a-specific-table)
+- [Storage usage](#storage-usage)
+  - [View the size of tables in megabytes](#view-the-size-of-tables-in-megabytes)
+  - [View the size of a specific tables in megabytes](#view-the-size-of-a-specific-tables-in-megabytes)
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/admin/cluster-data.md:237]:
```suggestion
  - [View the size of a specific table in megabytes](#view-the-size-of-a-specific-table-in-megabytes)
```

File: content/influxdb/cloud-dedicated/admin/cluster-data.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/admin/cluster-data.md:2]:
```suggestion
title: Query system data
```

File: content/influxdb/cloud-dedicated/admin/cluster-data.md [hunk 6]
```diff
@@ -0,0 +1,398 @@
+---
+title: Query cluster system data
+description: >
+  Query system tables in your InfluxDB Cloud Dedicated cluster to see data related
+  to queries, tables, partitions, and compaction in your cluster.
+menu:
+  influxdb_cloud_dedicated:
+    parent: Administer InfluxDB Cloud
+    name: Query cluster system data
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/admin/cluster-data.md:9]:
```suggestion
    name: Query system data
```

**Pull request merged**